bandit/plugins/yaml_load.py

```diff
@@ -55,7 +55,8 @@
 @test.test_id('B506')
 @test.checks('Call')
 def yaml_load(context):
-    if isinstance(context.call_function_name_qual, str):
+    if context.is_module_imported_exact('yaml') and \
```

Comment:
Please avoid using \ in long if statement lines.  Use ( ) instead

Reply:
done